src/components/NewTodoForm.js

```diff
@@ -0,0 +1,90 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import styled from 'styled-components';
+import {loadFromLocalStorage, writeToLocalStorage} from '../util/localstorage';
+
+export default function NewTodoForm({insertNewTodo}) {
+  const [task, setTask] = useState('');
+
+  const [category, setCategory] = useState(42);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const currentTemplates = loadFromLocalStorage('TaskTemplates');
+
+    const newTask = {id: nanoid(), task, checkedAt: '', isChecked: false};
+
+    writeToLocalStorage(
+      'TaskTemplates',
+      currentTemplates.map(template => {
+        if (template.tasks !== undefined) {
+          if (template.weekday === category) {
+            return {...template, tasks: [...template.tasks, newTask]};
+          }
+        }
+        return template;
+      })
+    );
+
+    insertNewTodo(category, newTask);
+  }
+
+  return (
+    <Wrapper>
+      <StyledForm onSubmit={handleSubmit}>
+        <label htmlFor="task">
+          Aufgaben:

```

Comment:
Do you want the user to input one or more tasks at once? The placeholder tells me it's only one ;)